component/src/main/ballerina/net.ftp/ftp_client_endpoint.bal

```diff
@@ -0,0 +1,55 @@
+package ballerina.net.ftp;
```

Comment:
License header?

Reply:
I have added to the new file. This local DirectoryListener module will move to the main ballerina repo.